OrbitService/Utils.cpp

```diff
@@ -121,32 +130,32 @@ ErrorMessageOr<std::vector<ModuleInfo>> ListModules(int32_t pid) {
   return result;
 }
 
-outcome::result<std::unordered_map<pid_t, double>> GetCpuUtilization() {
-  const std::string cmd = "top -b -n 1 | sed -n '8, 1000{s/^ *//;s/ *$//;s/  */,/gp;};1000q'";
-  OUTCOME_TRY(result, ExecuteCommand(cmd));
+ErrorMessageOr<std::unordered_map<pid_t, double>> GetCpuUtilization() {
+  const std::string cmd = "top -b -n 1 -w512 | sed -n '8, 1000{s/^ *//;s/ *$//;s/  */,/gp;};1000q'";
```

Comment:
this is slow and prone to top output change. but I out of scope for this PR. I think this is last command we use right? nothing else uses command-line utils?

Reply:
Not anywhere else on the Service as far as I am aware. I would also put this in a different PR. The parameter I added (-w512) already solved a weird bug that is related to terminal window size, which I encountered coincidentally.